bot.js

```diff
@@ -1017,6 +1019,8 @@ bot.on('message', function (user, userID, channelID, message, evt) {
 					let dispname = bot.users[uss].username
 					if (member.nick != null){
 						dispname = member.nick
+					} else {

```

Comment:
Very stupid here. The display name is originally their username. And now you want it to be "None" if there is no nickname? Bad.